src/exchanges/crdtExchange.ts

```diff
@@ -40,6 +42,7 @@ export const isDeadlockMutation = (error: undefined | CombinedError) => {
 
 export type CrdtExchangeOpts =  {
   isRetryableError: (res: OperationResult) => boolean;
+  storage: DefaultStorage,
```

Comment:
Can we skip straight to using the `StorageAdapter` interface from urql?

Reply:
Yeah, that makes sense.